src/modules/dashboard/components/ColonyActionsDialog/ColonyActionsDialogItem.css

```
@@ -0,0 +1,72 @@
+@value blueHover: rgb(76,93,255);
```

please add spaces between the r, g and b values

```suggestion
@value blueHover: rgb(76, 93, 255);
```